SubmittedAssignments/view-invoice-format2/index.html

```
@@ -0,0 +1,102 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Invoice Page</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+   
+    <invoice-component
+        config='{
```

HTMLHint (1.5.0)

[error] The value of attribute [ config ] must be in double quotes.

(attr-value-double-quotes)